src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -514,7 +514,19 @@ internal bool RuntimeSupportsCovariantReturnsOfClasses
         internal abstract ImmutableArray<AssemblySymbol> GetLinkedReferencedAssemblies();
         internal abstract void SetLinkedReferencedAssemblies(ImmutableArray<AssemblySymbol> assemblies);
 
+        IEnumerable<ImmutableArray<byte>> IAssemblySymbolInternal.GetInternalsVisibleToPublicKeys(string simpleName)
```

Comment:
Can we avoid adding private implementations by just making the existing abstract methods public?

Reply:
The existing patterns for these interface implementations is to have private implementations. I don't want to change that here.